### ISSUE TITLE
Only partial uploads ..

### ISSUE THREAD

**Issue by vamsiikrishna** [Altinity/clickhouse-backup#163]
Hi !

I was trying clickhouse-backup ( latest alpa release ) and I am having some issues with S3 Upload.

Here are the steps I am following to perform the backup:

backup 

BACKUP_NAME=my_backup_$(date -u +%Y-%m-%dT%H-%M-%S)
clickhouse-backup create $BACKUP_NAME
clickhouse-backup upload $BACKUP_NAME

while uploading , I am able to see the progress bar. 

```
158.87 MiB / 5.94 GiB [===>-------------------------------------------------------------------------------------------- 
2.61% 3s
```
After it completes around 2.61 % , the program stops and prints 

```
2021/03/10 11:24:40  info done                      backup=my_backup_2021-03-10T11-16-26 operation=upload

```

and in S3 only 158.87 MiB is uploaded. 

EDIT :

CH Server Version: 21.1.2

**Comment by ElPeque** [comment]
Is it working at all?

It looks to me like with gzip compression it is only uploading metadata but no real data. With none compression it seems to be uploading data.

Maybe I should stick to the v0.6.4 branch but it would suck to leave behind table filtered download/upload/restore and the 'none' compression. What is the overall stability of the alpha? Thank you!

**Comment by AlexAkulov** [comment]
Fixed in [1.0.0-beta2](https://github.com/AlexAkulov/clickhouse-backup/releases/tag/1.0.0-beta2)

**Comment by mastertheknife** [comment]
This looks like the issue i am seeing, with version 1.0.0-beta2: https://github.com/AlexAkulov/clickhouse-backup/issues/203

**Comment by AlexAkulov** [comment]
I have improved error handling on upload in aadce0b. Please try again with version from master.

**Comment by AlexAkulov** [comment]
Fixed [v1.0.0](https://github.com/AlexAkulov/clickhouse-backup/releases/tag/v1.0.0)
